packages/styled-components/src/models/ServerStyleSheet.js

```diff
@@ -110,4 +101,8 @@ export default class ServerStyleSheet {
 
     return readableStream.pipe(transformer);
   }
+
+  seal() {
+    this.sealed = true;
```

Comment:
doesn't really change anything as far as I can see, but just here for API backward compat